src/format.ts

```diff
@@ -15,11 +15,18 @@
  */
 import * as fs from 'fs';
 import * as path from 'path';
+import {promisify} from 'util';
+
 import {Options} from './cli';
 import {createProgram} from './lint';
 
 // Exported for testing purposes.
 export const clangFormat = require('clang-format');
+const xml2js = require('xml2js');
+const chalk = require('chalk');
```

Comment:
Can you use `import * as x from 'x'` instead of `const x = require('x')`? TS will prompt you for type definitions -- you can get them with

`npm install --save-dev @types/x`

The `@types/` packages contain type definitions for each package. Excluding xml and utf (because you're not using them), I believe the rest of them all have this associated package.

Reply:
done